docs/section6/Rule6-13.md

```diff
@@ -10,41 +10,69 @@
 - Table G3.7, Performance Rating Method Lighting Power Density Allowances and Occupancy Sensor Reductions Using the Space-by-Space Method  
 
 **Applicability:** All required data elements exist for P_RMR  
-**Applicability Checks:** None  
+**Applicability Checks:**  
+
+  1. Pass Rule 6-10.
+
 **Manual Check:** Yes  
 **Evaluation Context:** Each Data Element  
 **Data Lookup:** Table G3.7  
+**Function Call:**  
+
+  - compare_schedules()
+  - normalize_space_lighting_schedule()
 
 ## Rule Logic:  
 
-- For each building_segment in the Proposed model: ```for building_segment_p in P_RMR.building.building_segments:```  
+- **Applicability Check 1:** If pass Rule 6-10: `if rule_6_10 == TRUE:`  
+
+- Get building open schedule in the proposed model: `building_open_schedule_p = P_RMR.building.building_open_schedule`  
+
+- For each building segment in building: `for building_segment_p in P_RMR.building.building_segments:`  
+
+  - For each thermal block in building segment: `thermal_block_p in building_segment_p.thermal_blocks:`  
+
+    - For each zone in thermal block: `zone_p in thermal_block_p.zones:`  
+
+      - For each space in zone: `space_p in zone_p.spaces:`  
+
+        - Get matching space from B_RMR: `space_b = match_data_element(B_RMR, Spaces, space_p.id)`  
+
+          - Get normalized space lighting schedule for B_RMR: `normalized_schedule_b = normalize_space_lighting_schedule(space_b)`  
+
+        - Get lighting space type: `lighting_space_type_p = space_p.lighting_space_type`  
+
```

Comment:
need normalized schedule at space level for proposed as well. Proposed could have several lighting objects with differant schedules and we should be calculating the normalized schedule at the space level for P as well.

Reply:
Agreed

Reply:
Right now the RDS compared each lighting object schedule with baseline. This won't work when baseline has multiple lighting objects with different schedule and needs to be modified. 
I'm not sure if we need to normalize schedules in P_RMR (using the normalize schedule function we have right now). For example, if 
B_RMR:
- LPD1 = 1W/sf, schedule = 0.5
- LPD2 = 1W/sf, schedule = 0.8, (assuming B_RMR multiple LPD should be the same, probably with the exception of retail lights)
- Total LPD = 2W/sf, normalized schedule = 0.65

P_RMR if no controls:
- LPD1 = 0.5W/sf, schedule = 0.5
- LPD2 = 0.3W/sf, schedule = 0.8
- Total LPD = 0.8W/sf, normalized schedule = 0.6125
In this case, averaging all schedules then compare to the normalized baseline schedule will work, with the exception when multiple lighting objects in B_RMR have different LPDs.

P_RMR with occ sensor only:
- LPD1 = 0.5W/sf, schedule = 0.4 (assuming occ sensor reduction factor of 0.2)
- LPD2 = 0.3W/sf, schedule = 0.72 (assuming occ sensor reduction factor of 0.1)
- Total LPD = 0.8W/sf, normalized schedule = 0.52
In this case, we can calculate each lighting schedule without reduction factor and average all schedules then compare to the normalized baseline schedule. Again it won't seem to work if multiple lighting objects in B_RMR have different LPDs.

P_RMR with both occ sensor and other type of controls:
In this case maybe we just compare if the normalized schedule in P_RMR is less than B_RMR.

Or do I need to modify the normalize schedule function?

Reply:
I think the baseline space lighting may include several interior lighting objects with different LPDs. For example, general and task lighting in a space may be modeled as separate interior lighting objects each with a different schedule. This would meet App G requirements as long as the total LPD equials the expected value in Table G3.7.  

To make this RDS work, it seems that we need to require that baseline either (a) has a single interior lighting object, or (b) has the same number of lighting objects as the proposed design. If (a), we'd compare each of the proposed schedules to the single baseline schedule individually, and confirm that they are either the same (no controls) or the difference is as expected. If (b), we'd expect pairs of interior lighting objects and compare schedules to establish whether they relate to each other as expected.

Reply:
@xingcx agree with MK. there is no requirement/guidance in App G for the B to have multiple lighting objects. In most cases, we would see the B RMR with a single lighting object with the prescribed LPD based on Table G3.7. The most conservative option would be to use the normalized schedules for both B and P. We can also discuss this tomorrow if you think otherwise.

Reply:
@mkarpman you are right, if B_RMR has the same allocation as P_RMR, normalizing LPD at the space level works. I'll update the rule.